src/Stratis.Bitcoin.Features.PoA/Voting/PollsRepository.cs

```diff
@@ -153,6 +151,16 @@ public void Initialize()
                                 modified = true;
                             }
 
+                            // Check if the current tip is before the poll expiry activation block,
+                            // if so un-expire it.
+                            if (poll.IsExpired && this.CurrentTip.Height < this.network.ConsensusOptions.Release1100ActivationHeight)
```

Comment:
@quantumagi this part is important to review.

Reply:
```
if (poll.IsExpired && !PollIsExpiredAt(poll, this.CurrentTip, this.network))
{
  poll.IsExpired = false;
  ...
}

public static bool IsPollExpiredAt(Poll poll, ChainedHeader chainedHeader, Network network)
{
    if (chainedHeader == null)
        return false;

    return Math.Max(poll.PollStartBlockData.Height + network.ConsensusOptions.PollExpiryBlocks, network.ConsensusOptions.Release1100ActivationHeight) <= chainedHeader.Height;
}
```

Reply:
@quantumagi fixed, please re-check.